GSA/openacr-editor
# Handling links in chapter help text

Closes https://github.com/GSA/openacr/issues/288 and closes https://github.com/GSA/openacr/issues/290

- Created a function to open first accordion for anchor links in chapter help text.
- Stopped the alert (You are about to lose content) from showing when users click on the internal, but different route link in the 'software' help text.

## QA
Note: Test in multiple browsers and please comment re: which browsers tested.

- After deployment ('Deploy' action) is successful, open site https://gsa.github.io/openacr-editor/.
- Confirm that chapter help text is displaying properly (correct text for the correct chapter and is well-formatted)
- For A and AA, test the links in the chapter help text. Confirm that if a link is clicked, the first accordion on the page opens. 
- For Software, click the link in the chapter help text. Confirm that you navigate to A without getting the alert. 
- Return to the 'Software' page and modify an input field. Then click the link. Confirm that you navigate to A without getting the alert. Next navigate to an external url, confirm that you DO get the alert at that point
